s19/custom_mixin.rb

```diff
@@ -0,0 +1,42 @@
+module Purchaseable
```

Comment:
What's the difference between a `class` and a `module`?

Reply:
Modules are only a set of methods and instances cannot be extracted from them.